### DISCUSSION TITLE
Starrocks Integration

### DISCUSSION THREAD

**Discussion by SaurabhBaranawal** [prisma/prisma#23760] [Q&A]
### Question

I want to connect to my starrocks database from prisma. But upin trying to connect using a mysql provider i am getting error 

Schema engine error:
Getting analyzing error. Detail message: Unknown system variable 'socket', the most similar variables are {'hdfs_backend_selector_hash_algorithm', 'use_compute_nodes', 'max_scan_key_num'}.

### How to reproduce (optional)

Just use any prisma command

**Comment by janpio** [comment]
Seems like this database does not support the same functionality we expect to be supported. What `provider` value are you using for this database?

**Reply by huben92** [reply]
Im also having the same issue, using mysql connector

**Comment by rainems** [comment]
I would also love to use prisma with starrocks. Has anyone figured out a work around to do introspection?

**Comment by SaurabhBaranawal** [comment]
I am able to connect to starrocks using Mysql connector

**Reply by huben92** [reply]
> I am able to connect to starrocks using Mysql connector
> […](#)
> On Mon, 17 Jun, 2024, 4:10 pm huben92, ***@***.***> wrote: Im also having the same issue, using mysql connector — Reply to this email directly, view it on GitHub <[#23760 (reply in thread)](https://github.com/prisma/prisma/discussions/23760#discussioncomment-9793088)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AK6RHLS3ZKKBTUZJ6KMIDLTZH24KXAVCNFSM6AAAAABF4ELGTCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOOJTGA4DQ> . You are receiving this because you authored the thread.Message ID: ***@***.***>

What prisma version and starrocks version you using sir?

**Comment by SaurabhBaranawal** [comment]
I do not use prisma. I am directly connected using Mysql connector

**Reply by huben92** [reply]
Yes im able to connect to starrocks with mysql connector without any problem, but i can't connect to starrocks using prisma.
I can't use another any other orm, prisma too good

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
